src/app/App.js

```diff
@@ -79,6 +83,12 @@ function renderNavMenu(parent, activeItemNr = 0) {
   parent.appendChild(comp);
 }
 
+function renderGameMode(parent) {
+  const comp = GameMode();
```

Comment:
Co jeśli chciałbym wyrenderować GameMode z innym tekstem. W powyższym przykładzie nie jest to możliwe

Reply:
up

Reply:
?

Reply:
Jak dla mnie trzeba właśnie przerobić tę funkcję render na taką uniwersalną, jak pisałem niedawno - bo ta tutaj właśnie sprawia, że każdy komponent musiałby mieć inną i problem np. właśnie z parametrami.

Reply:
Czyli się zgadzamy?